src/main/java/picard/nio/GATKBucketUtils.java

```diff
@@ -0,0 +1,160 @@
+package picard.nio;
```

Comment:
@cmnbroad @droazen @lbergelson these files were "ported" from GATK in order to make temporaryFiles for tests